app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/agenda/local/entity/AgendaEntity.kt

```diff
@@ -0,0 +1,13 @@
+package com.davidtomas.taskyapp.features.agenda.data.agenda.local.entity
+
+import com.davidtomas.taskyapp.features.agenda.data.event.local.entity.AttendeeEntity
+import com.davidtomas.taskyapp.features.agenda.data.event.local.entity.PhotoEntity
+import io.realm.kotlin.ext.realmListOf
+import io.realm.kotlin.types.RealmList
+import io.realm.kotlin.types.RealmObject
+
+class AgendaEntity : RealmObject {
```

Comment:
Why would you need to save the agenda as entity? What even is one agenda? All agenda items? Just those of a single day? That's not very clear